src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.cs

```diff
@@ -149,6 +154,12 @@ protected override Task OnClientConnectedAsync(OpenConnectionMessage message)
         connection.ServiceConnection = this;
 
         connection.Features.Set<IConnectionMigrationFeature>(null);
+
+        if (!_cultureInfoManager.TryApplyCulture(connection.RequestId))
```

Comment:
```cs
if (_cultureInfoManager.TryGetCultureFeature(connection.RequestId, out var feature)){
connection.GetHttpContext().Features.Set<IRequestCultureFeature>(feature)
}

```

Reply:
`connection.GetHttpContext().Features.Set<IRequestCultureFeature>(feature)` doesn't take effect
I use `CultureInfo.CurrentCulture=XXX` and `CultureInfo.CureentUICulture=XXX`